test/serde/serde_helpers.py

```diff
@@ -1374,48 +1416,67 @@ def compare(detailed, original):
 # syft.messaging.message.CommandMessage
 def make_command_message(**kwargs):
     bob = kwargs["workers"]["bob"]
+    alice = kwargs["workers"]["alice"]
     bob.log_msgs = True
 
     x = torch.tensor([1, 2, 3, 4]).send(bob)
     y = x * 2
-    op1 = bob._get_msg(-1)
+    cmd1 = bob._get_msg(-1)
 
     a = torch.tensor([[1, 2], [3, 4]]).send(bob)
     b = a.sum(1, keepdim=True)
-    op2 = bob._get_msg(-1)
+    cmd2 = bob._get_msg(-1)
+
+    x = torch.tensor([1, 2, 3, 4]).send(bob)
+    x.remote_send(alice)
+    cmd3 = bob._get_msg(-1)
 
     bob.log_msgs = False
 
     def compare(detailed, original):
-        assert type(detailed) == syft.messaging.message.CommandMessage
-
-        detailed = detailed.action
-        original = original.action
-
-        detailed_msg = (detailed.name, detailed.target, detailed.args, detailed.kwargs)
-        original_msg = (original.name, original.target, original.args, original.kwargs)
-        for i in range(len(original_msg)):
-            if type(original_msg[i]) != torch.Tensor:
-                assert detailed_msg[i] == original_msg[i]
-            else:
-                assert detailed_msg[i].equal(original_msg[i])
-        assert detailed.return_ids == original.return_ids
-        return True
+        if isinstance(detailed.action, syft.execution.computation.ComputationAction):
```

Comment:
👍 

Would be nice if the `compare` functions were re-usable outside the scope of the original `make_whatever` function, but that's definitely out of scope for this PR.